### DISCUSSION TITLE
Законность использования этого API

### DISCUSSION THREAD

**Discussion by rodion-m** [MarshalX/yandex-music-api#457] [Q&A] [answered]
Добрый день! Спасибо большое за ваш труд, но интересует вопрос законности использования этого апи. Где-то это прописано?

**Comment by MarshalX** [comment, marked as answer]
Добрый день. Можете почитать вот сюда и доразобраться в теме https://habr.com/ru/post/487428/#comment_21248764

**Reply by rodion-m** [reply]
Спасибо!
